.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       matrix:
         go: ['1.14', '1.15']
         os: ['macos-10.15']
-        xcode-version: ['12.0', '11.7', '11.6', '11.5', '11.4.1', '11.3.1', '11.2.1', '11.1', '11.0']
+        xcode-version: ['12.1', '12.0', '11.7', '11.6', '11.5', '11.4.1', '11.3.1', '11.2.1']
```

Comment:
Should it be

12.0.1 instead of 12.0?
Can we also include 12.2 as a non-mandatory check?

Reply:
I tried 12.0.1, but it failed somehow: https://github.com/google/pprof/runs/1398239582?check_suite_focus=true

Error: Could not find Xcode version that satisfied version spec: '12.0.1'

Added 12.2. Could you update the check requirements in the settings? Thanks!

Reply:
I marked 12.1 as required. It shows 12.0.1 in the list FWIW.